public/static/locales/en-US/landing.json

```diff
@@ -22,6 +22,13 @@
   },
   "credits": "MySK is a project by SK IT Solutions • Supervised by Supannee Supeerath and Atipol Sukrisadanon • Design and development by Siravit Phokeed and Smart Wattanapornmongkol • Quality assurance by Tempoom Leelacharoen • Translations to Thai contributed by Suankularb English Club",
   "dialog": {
+    "accountNotFound": {
+      "title": "Account not found",
+      "desc": "There might be a problem with your account. If you are a current student, please report this issue via <0>this form</0>.",
```

Comment:
Just asking, what is this <0> element here?

Reply:
[Replacement tags](https://react.i18next.com/latest/trans-component#how-to-get-the-correct-translation-string). `next-i18next` will use the first (0-index) component passed into `Trans` to replace `<0></0>`.